File: p09_Number_Triangle/pattern.java
import java.util.Scanner;

public class pattern{
    public static void main(String[] args){
        Scanner scan = new Scanner(System.in);
        
        int row = scan.nextInt();

        for(int i = 1; i <= row; i++){
            String rowNum = Integer.toString(i).repeat(i);
            System.out.println(rowNum);
        }

        scan.close();
    }
}

File: p09_Number_Triangle/pattern.cpp
#include<iostream>
#include<string>

using namespace std;

int main(){
    int row;
    cin >> row;

    for(int i = 1; i <= row; i++){
        char rowNum = i + '0';
        string num = string(i, rowNum);

        cout << num << endl;
    }

    return 0;
}